lms/services/vitalsource/_client.py

```diff
@@ -96,6 +97,22 @@ def get_table_of_contents(self, book_id: str) -> List[dict]:
 
         return toc
 
+    def get_courses(self):
```

Comment:
If I remember correctly of off this course and roster fetching was needed to get the ID of the student in the VS system.